### DISCUSSION TITLE
Release cadence/process

### DISCUSSION THREAD

**Discussion by dfields-msft** [dotnet/Silk.NET#954] [General] [answered]
I perused the READMEs but didn't find an answer to this - what drives the release process, and what is the cadence? I noticed that there was no May release; when will the June release happen? I'm interested because there are some recent additions to OpenXR which I'd like to pick up, and I assume that next time the release crank turns, these will be incorporated into Silk.NET. I could clone the repo and build from source, but I'd rather just pick up a fresh NuGet if it will be available soon. Thanks!

**Comment by HurricanKai** [comment, marked as answer]
Hey! We've felt that Silk.NET 2.x is currently rather stable and have stopped providing monthly releases. We'll likely release 2.16 in the coming week, which of course will include all the latest bindings.
We've never communicated this via GitHub, so no worries, I'll update the README asap

**Reply by dfields-msft** [reply]
Thanks, this is helpful! I might argue that even if the tooling is stable/not churning, to the extent that the source APIs continue to be updated, it would be helpful to keep the crank turning periodically so 2.x doesn't become stale. I suppose that depends how close the project is to fully transitioning to 3.0 as well.

**Reply by HurricanKai** [reply]
Periodically updating 2.x would certainly be good, it's quite a bit of work though, so I don't want to commit to doing it every month, so it's just a best-effort update now (primarily when relevant updates happen).
3.0 is quite a bit out, but what I am spending most of dev resources right now.
